Dockerfile

```diff
@@ -28,10 +28,12 @@ RUN dnf -y --disableplugin=subscription-manager --setopt=tsflags=nodocs install
       mod_intercept_form_submit \
       mod_lookup_identity \
       mod_auth_mellon && \
-    dnf -y --disableplugin=subscription-manager --setopt=tsflags=nodocs install \
-      http://mirror.centos.org/centos/8-stream/BaseOS/${ARCH}/os/Packages/centos-stream-repos-8-2.el8.noarch.rpm \
-      http://mirror.centos.org/centos/8-stream/BaseOS/${ARCH}/os/Packages/centos-gpg-keys-8-2.el8.noarch.rpm && \
-    dnf -y --disableplugin=subscription-manager module enable mod_auth_openidc && \
+    dnf -y --setopt=protected_packages= remove redhat-release && \
```

Comment:
Is this clearing the protected packages so you can remove redhat-release and subsubscription manager or one of them?

Reply:
Yes